numpy/core/tests/test_api.py

```diff
@@ -38,58 +38,39 @@ def test_array_array():
         np.array(np.array(1.0, dtype=np.float64), dtype=np.float64)
         assert_equal(old_refcount, sys.getrefcount(np.float64))
 
+    # test string
+    B2 = np.dtype((bytes, 2))
+    B3 = np.dtype((bytes, 3))
+    B5 = np.dtype((bytes, 5))
```

Comment:
```suggestion
    S5 = np.dtype((bytes, 5))
```
Since that's what `str(S5)` gives

Reply:
Took me a while to understand this comment.
```
>>> np.dtype((bytes, 5))
dtype('S5')
```